packages/kit/src/core/preview/index.js

```diff
@@ -161,7 +161,7 @@ export async function preview({ port, host, config, https: use_https = false })
 	});
 
 	return new Promise((fulfil) => {
-		http_server.listen(port, host || '0.0.0.0', () => {
+		http_server.listen(port, host, () => {
```

Comment:
Just to confirm: When the user passes `--host` with no argument (which indicates they want to bind on all interfaces), `host` will be an empty string, and passing `''` to `.listen` will have that effect?

Reply:
Passing `''` would be equivalent to `undefined` and results in `0.0.0.0` or `::`, depending on the network support (and I just tested to confirm it). The difference in this case is that we will bind all ipv6 and ipv4 interfaces, whereas before we only did ipv4.

Reply:
:+1: Sounds good. I think that warrants a changeset, and then this is ready to go!